docs/How_To_Update_ONNX_Dev_Notes.md

```diff
@@ -18,15 +18,12 @@ This file should be generated. See [cgmanifests/README](/cgmanifests/README.md)
 3. Update [tools/ci_build/github/linux/docker/scripts/install_onnx.sh](/tools/ci_build/github/linux/docker/scripts/install_onnx.sh).
```

Comment:
this is not valid any more. Instead we need to update the 2 requirements.txt
https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/linux/docker/scripts/manylinux/requirements.txt
and
https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/linux/docker/scripts/requirements.txt